localstack/services/apigateway/helpers.py

```diff
@@ -108,12 +119,14 @@ def get_apigateway_store(account_id: str = None, region: str = None) -> ApiGatew
 
 
 class Resolver:
-    def __init__(self, document: dict, allow_recursive=True):
+    def __init__(self, document: dict, rest_api_id: str, allow_recursive=True):
         self.document = document
         self.allow_recursive = allow_recursive
         # cache which maps known refs to part of the document
         self._cache = {}
         self._refpaths = ["#"]
+        host_definition = localstack_host(use_localhost_cloud=True)
```

Comment:
never seen this before, what is this?

Reply:
I found the PR introducing it: https://github.com/localstack/localstack/pull/7774